test/test-sample-app.js

```diff
@@ -61,9 +61,13 @@ describe('Static translation of an app with two locale directories', function()
 
   it('translates the title in the json data into French', function() {
     var content = String(fs.readFileSync(appPath + '/build/fr/data.json'));
-    expect(content).to.equal('{"title":"Bonjour tout le monde"}');
+    expect(content).to.contain('"title":"Bonjour tout le monde"');
   });
 
+  it('prefixes the url in the json data with french language code', function() {
```

Comment:
can you use Portuguese here? Then we'll get some test coverage on the lang prefixing